Lib/typing.py

```diff
@@ -823,6 +874,96 @@ def __init__(self, name, *constraints, bound=None,
             self.__module__ = def_mod
 
 
+class TypeVarTuple(_Final, _Immutable, _root=True):
```

Comment:
I was hoping I could steal a docstring for TypeVarTuple from you for use in typing-extensions :) A docstring should be added here.